django_elasticsearch_dsl/__init__.py

```diff
@@ -4,18 +4,25 @@
 from .indices import Index  # noqa
 from .signals import delete_document, update_document # noqa
 from .fields import (  # noqa
-    StringField,
-    FloatField,
-    DoubleField,
+    AttachmentField,
+    BooleanField,
     ByteField,
-    ShortField,
+    CompletionField,
+    DateField,
+    DoubleField,
+    FloatField,
+    GeoPointField,
+    GeoShapeField,
     IntegerField,
+    IpField,
+    KeywordField,
+    ListField,
     LongField,
-    DateField,
-    BooleanField,
-    ObjectField,
     NestedField,
-    ListField,
+    ObjectField,
+    ShortField,
+    StringField,
+    TextField,
```

Comment:
It is not backward compatible with `elasticsearch-dsl>=2.0.0,<3.0.0`. Maybe we can just add a try catch for the TextField and KeywordField